src/App.jsx

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import Calculator from "./components/Calculator";
+
+const App = () => {
+  return <Calculator />;
+};
+
```

Comment:
- [ ] Kindly make the App a class based component, as per [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Use%20class%20based%20components%20(All%20components%20should%20be%20class%2Dbased).) requirement.

Reply:
Hi @ShadyShawkat, thanks for your feedback! :+1: